### ISSUE TITLE
Feature request: Allow configuration management in mix.exs 

### ISSUE THREAD

**Issue by asummers** [zachdaniel/git_ops#28]
Various other tools require knowledge of the git repo, e.g. `ex_doc` in `mix.exs` and also allow configuration via the `mix.exs` file. [You have](https://github.com/zachdaniel/git_ops/blob/master/mix.exs#L16) it [twice](https://github.com/zachdaniel/git_ops/blob/master/mix.exs#L42) (see also the typo in the first link :P) Allowing configuration here would be one fewer spot where a name of the git repo or even the app name itself would need to be propagated to. Additionally, several internal libraries do not even have config files except for `git_ops` which feels really strange on a large enough number of projects.

**Comment by zachdaniel** [comment]
Yeah, I think it would be reasonable to check there first. We should probably allow it to be overridden in the gitops config Still though.
